app/components/gallery/ListItem.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="aspect-ratio-1/1">
+    <NuxtLink :to="gallery.slug" class="relative gallery-preview preload-block">
+      <img :src="previewImage.full_image_url" :srcset="previewImage.medium_srcset" :alt="htmlDecoder(gallery.title.rendered)">
+      <div class="absolute bottom-0 flex flex-col justify-end w-full text-center text-white gallery-title">
+        <h3 class="text-lg">{{ htmlDecoder(gallery.title.rendered) }}</h3>
+      </div>
+    </NuxtLink>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    gallery: {
+      type: Object,
+      required: true
+    }
+  },
+  computed: {
+    previewImage () {
+    /* TODO -- check if featured image, if not fallback to first image */
```

Comment:
I think it would be perfectly fine like this if we have a fallback default img.

But of course in general we try to add feature images anyway